bears/css/PostCSSBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Setting import typed_list
+
+
+class PostCSSBear(Lint, LocalBear):
+    executable = "postcss"
+    diff_message = "PostCSS plugins in action."
+    gives_corrected = True
+
+    def run(self, filename, file, postcss_plugins: typed_list(str)=[]):
+        '''
+        This bear adds ``PostCSS`` utilities to analyze CSS code.
+
+        :param postcss_plugins:     List of PostCSS plugins to use with
+                                    this bear which can be defined in
+                                    .coafile.
+        '''
+        if postcss_plugins:
+            for plugin in postcss_plugins:
+                self.prerequisite_command = ['postcss', '--use ', plugin]
+                self.prerequisite_fail_msg = "Plugin {} not installed".format(
+                    plugin)
+                self.arguments += (" --use "+plugin)
+            self.arguments += " {filename}"
+            return self.lint(filename, file)
+        else:
+            self.err("Please specify at least one plugin name.")
```

Comment:
> The tool itself is a Node.js module that parses CSS into an abstract syntax tree (AST); passes that AST** through any number of “plugin” functions;** and then converts that AST back into a string, which you can output to a file. Each function the AST passes through may or may not transform it; sourcemaps will be generated to keep track of any changes.
The AST provides a straightforward API that developers can use to write plugins

http://davidtheclark.com/its-time-for-everyone-to-learn-about-postcss/

So yeah, without plugins I don't think it can do anything. Much like coala-bears doesn't do anything without specifying a bear.

![postcss](https://cloud.githubusercontent.com/assets/5689132/13980110/a7d48c58-f102-11e5-8aba-bd27f9655f31.png)